code/game/objects/items/weapons/tools/shovel.dm

```diff
@@ -17,32 +17,35 @@
 
 /obj/item/weapon/tool/shovel/improvised
 	name = "junk shovel"
-	desc = "A large but fragile tool for moving dirt and rock."
+	desc = "A large but fragile tool for moving dirt and rock, made by hand. Has more then enough space for tool mods to make it better."
```

Comment:
"A large but fragile tool for moving dirt and rock, made by hand. Has more th**a**n enough space for tool mods to make it better."